bundles/eclipselink/src/main/java/module-info.java

```diff
@@ -45,6 +45,8 @@
 
     requires static com.sun.xml.bind.core;
 
+    uses org.eclipse.persistence.internal.databaseaccess.spi.JsonPlatformProvider;
```

Comment:
the flow of instructions in other descriptors is `requires`, `exports` and then `uses`. What is the reason for breaking the consistency here?

Reply:
see ie https://dev.java/learn/introduction-to-modules-in-java/